app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -37,7 +42,14 @@ def convert_to_group_trigger(miq_alert)
       when "mw_accumulated_gc_duration"
         context = { 'dataId.hm.type' => 'counter', 'dataId.hm.prefix' => 'hm_c_' }
       end
-      ::Hawkular::Alerts::Trigger.new('id'          => "MiQ-#{miq_alert[:id]}",
+
+      trigger_id = if operation == :new
+                     @ems.miq_id_prefix("alert-#{miq_alert[:id]}")
```

Comment:
Yes, that's how IDs will look.
And yes, integration tests aren't trivial. But it may not be too hard to do what you are asking for. I'll try.